File: apps/s-2-admin/src/api/bookingReference/BookingReferenceWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringFilter } from "../../util/StringFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";
import { BooleanNullableFilter } from "../../util/BooleanNullableFilter";

export type BookingReferenceWhereInput = {
  id?: IntFilter;
  typeField?: StringFilter;
  uid?: StringFilter;
  meetingId?: StringNullableFilter;
  meetingPassword?: StringNullableFilter;
  meetingUrl?: StringNullableFilter;
  booking?: BookingWhereUniqueInput;
  externalCalendarId?: StringNullableFilter;
  deleted?: BooleanNullableFilter;
};

File: apps/cactus-demo/src/APIs/BookingReference/Dtos/BookingReferenceWhereInput.cs
namespace CactusDemo.APIs.Dtos;

public class BookingReferenceWhereInput
{
    public int? Id { get; set; }

    public string? TypeField { get; set; }

    public string? Uid { get; set; }

    public string? MeetingId { get; set; }

    public string? MeetingPassword { get; set; }

    public string? MeetingUrl { get; set; }

    public string? ExternalCalendarId { get; set; }

    public bool? Deleted { get; set; }

    public int? Booking { get; set; }
}
